src/components/GettingStarted/GettingStarted.js

```diff
@@ -3,62 +3,143 @@ import RoutePath from 'lib/routePath';
+import SimpleExample from './Steps/SimpleExample';
+
+const GettingStarted = (props) => {
+  const pathParams = {
+    orgId: props.match.params.orgId,
+    clusterId: props.match.params.clusterId,
+  };
+
+  const clusterGuideConfigurationPath = RoutePath.createUsablePath(
+    OrganizationsRoutes.Clusters.GettingStarted.ConfigureKubeCtl,
+    pathParams
+  );
+
+  const clusterGuideExamplePath = RoutePath.createUsablePath(
+    OrganizationsRoutes.Clusters.GettingStarted.SimpleExample,
+    pathParams
+  );
+
+  const clusterGuideIngressPath = RoutePath.createUsablePath(
+    OrganizationsRoutes.Clusters.GettingStarted.InstallIngress,
+    pathParams
+  );
+
+  const clusterGuideNextStepsPath = RoutePath.createUsablePath(
+    OrganizationsRoutes.Clusters.GettingStarted.NextSteps,
+    pathParams
+  );
+
+  const steps = [
+    {
+      title: 'Get Access',
+      description:
+        'Enable your Kubernetes CLI to access your Kubernetes cluster at Giant Swarm',
+      url: clusterGuideConfigurationPath,
+      routePath: OrganizationsRoutes.Clusters.GettingStarted.ConfigureKubeCtl,
+      component: ConfigureKubeCTL,
+    },
+
+    {
+      title: 'Run a simple example',
+      description:
+        'To make sure everything works as expected, let&apos;s start a hello world application',
+      url: clusterGuideExamplePath,
+      routePath: OrganizationsRoutes.Clusters.GettingStarted.SimpleExample,
+      component: SimpleExample,
+    },
+
+    {
+      title: 'Next steps',
+      description:
+        'We point you to some useful next best actions, like setting up the Kubernetes dashboard',
+      url: clusterGuideNextStepsPath,
+      routePath: OrganizationsRoutes.Clusters.GettingStarted.NextSteps,
+      component: NextSteps,
+    },
+  ];
+
+  // Insert the install ingress step if the cluster has optional ingress.
+  const ingressStep = {
+    title: 'Install an ingress controller',
+    description:
+      "We'll need an ingress controller before we can access any services from the browser",
+    url: clusterGuideIngressPath,
+    routePath: OrganizationsRoutes.Clusters.GettingStarted.InstallIngress,
+    component: InstallIngress,
+  };
+
+  if (props.hasOptionalIngress) {
+    steps.splice(1, 0, ingressStep);
+  }
+
+  return (
+    <DocumentTitle title='Getting Started'>
+      <Breadcrumb
+        data={{
+          title: 'GETTING STARTED',
+          pathname: RoutePath.createUsablePath(
+            OrganizationsRoutes.Clusters.GettingStarted.Overview,
+            {
+              clusterId: props.match.params.clusterId,
+              orgId: props.match.params.orgId,
+            }
+          ),
+        }}
+      >
+        <div>
+          <Switch>
+            <Route
+              render={() => <Overview steps={steps} {...props} />}
+              exact
+              path={OrganizationsRoutes.Clusters.GettingStarted.Overview}
+            />
+            {steps.map(({ component, routePath, title }, i) => (
```

Comment:
```suggestion
            {steps.map(({ component: Component, routePath, title }, i) => (
```

If we destructure the component like this, then we can use it like a regular component (ex: `<Component {...someProps} />`), instead of relying on `React.createElement`

Reply:
Here's a doc about it, for more context: https://reactjs.org/docs/jsx-in-depth.html#user-defined-components-must-be-capitalized